_posts/2020-07-23-what-was-one-crisis-is-now-policy.md

```diff
@@ -0,0 +1,29 @@
+layout : blog
+
+title : “What Was Once Crisis Is Now Policy”
+
+description : The pandemic made you adapt and adlib. What stuck?
+
+image : 2020-07-23-what-was-once-crisis-is-now-policy.jpg
+
+author : John Scott
+
+comments : true
+
+tags : [John Scott, Software,COVID-19]
+---
+
+<p>For most of us, we're at week 18 of this global health battle. It feels like it's been forever.</p>
+<p>We likely initially panicked, then furloughed/laid off people. We scrubbed everything in sight. Some of got a mask. Some of us didn't. We cut back, slimmed down, or maybe even staffed up. My wife works for an EdTech firm, deploying distance learning software solutions for K-12. She can barely keep up.</p>
+<p>We adapted to the market conditions, through expansion, contraction, or some other word created by necessity, the need for some of us to literally survive.</p>
+<p>Shutowns, lockdowns, inside, outside, six feet of this and that. In California, we have been through multiple iterations of locking this and shutting that. It's been very frustrating, and some Californians are taking their rage out on Governor Gavin Newsom. We're certain he's getting no joy from the moving goalposts, but as of the time of this writing we're looking at more than 11,000 new cases each day. We're a hot zone, especially in Los Angeles County.</p>
+<p>Businesses are scrambling, spending precious capital to adapt, and hoping for some stability. The one thing we don't have much of are things reliable and expected and constant. Not just in my home state but across the country, in your county and town.</p>
+<p>You made changes to your customer approach, your business model, and perhaps even your technology. One thing I noticed right away - the touchless menu. My friends own a restaurant. They didn't want to deal with menus and all those hands on them, so they employ a QR code a diner scans when they arrive. Up comes the menu and drinks/wine. A masked server takes our ouder, then delivers our meal to the table. We pay through an app and we're out. The restaurant has a safe flow, proper spacing too. We feel safe there and of course we want to support them and other local small businesses. </p>
```

Comment:
I think "so they employ a QR code a diner scans" should be "so they employ a QR code that a diner scans".